qa/rspec/commands/redhat.rb

```diff
@@ -29,7 +29,7 @@ def installed?(hosts, package)
         stdout = cmd.stdout
       end
       stdout.match(/^Installed Packages$/)
-      stdout.match(/^logstash.noarch/)
+      stdout.match(/^logstash.noarch/) || stdout.match(/^#{architecture_extension}/)
```

Comment:
since we currently match `^logstash.noarch`, the `architecture_extension` will have values such as `x86_64`, shouldn't it be `^logstash.#{architecture_extension}/` or something similar?

Reply:
right!